types/lib/model.d.ts

```diff
@@ -749,7 +754,7 @@ export interface BulkCreateOptions<TAttributes = any> extends Logging, Transacti
   individualHooks?: boolean;
 
   /**
-   * Ignore duplicate values for primary keys? (not supported by postgres)
+   * Ignore duplicate values for primary keys?
```

Comment:
While the `ignoreDuplicates` constant is undefined for postgres, the option maps directly to `onConflictDoNothing` https://github.com/sequelize/sequelize/blob/main/lib/dialects/abstract/query-generator.js#L197 which is defined for postgres https://github.com/sequelize/sequelize/blob/main/lib/dialects/postgres/index.js#L48